source/UIAHandler/__init__.py

```diff
@@ -1144,8 +1144,17 @@ def _isUIAWindowHelper(self, hwnd: int, isDebug=False) -> bool:  # noqa: C901
 			# Using IAccessible for NetUIHWND controls causes focus changes not to be reported
 			# when the ribbon is collapsed.
 			# Testing shows that these controls emits proper events but they are ignored by NVDA.
-			isOfficeApp = appModule.productName.startswith(("Microsoft Office", "Microsoft Outlook"))
-			isOffice2013OrOlder = int(appModule.productVersion.split(".")[0]) < 16
+			# this is not necessarily an office app, or an app with version information, for example geekbench 6.
+			try:
+				isOfficeApp = appModule.productName.startswith(("Microsoft Office", "Microsoft Outlook"))
+				isOffice2013OrOlder = int(appModule.productVersion.split(".")[0]) < 16
+			except RuntimeError:

```

Comment:
Shouldn't this warning be here?

```suggestion
			try:
				isOfficeApp = appModule.productName.startswith(("Microsoft Office", "Microsoft Outlook"))
				isOffice2013OrOlder = int(appModule.productVersion.split(".")[0]) < 16
			except RuntimeError:
				# this is not necessarily an office app, or an app with version information, for example geekbench 6.
```